mesheryctl/internal/cli/root/config/config.go

```diff
@@ -138,6 +140,38 @@ func (mc *MesheryCtlConfig) SetCurrentContext(contextName string) (Context, erro
 	return currCtx, err
 }
 
+// GetTokenForContext takes in the contextName and returns the token name and path corresponding
+// to the given current context
+func (mc *MesheryCtlConfig) GetTokenForContext(contextName string) (Token, error) {
+	ctx, ok := mc.Contexts[contextName]
+	if !ok {
+		return Token{}, fmt.Errorf("")
+	}
+
+	for _, t := range mc.Tokens {
+		if t.Name == ctx.Token {
+			return t, nil
+		}
+	}
+
+	return Token{}, fmt.Errorf("no token found for the given context")
+}
+
+func (t *Token) GetLocation() string {
+	if filepath.IsAbs(t.Location) {
+		return t.Location
+	}
+
+	// If file path is not absolute then assume that the file
+	// is in the .meshery directory
+	home, err := os.UserHomeDir()
+	if err != nil {
+		log.Warn("failed to get user home directory")
+	}
+
+	return filepath.Join(home, ".meshery", t.Location)
```

Comment:
@metonymic-smokey do we have a central constant for the application folder `.meshery` that should be referenced instead?

Reply:
We have one in utils.

Reply:
Yep, in `helpers.go` iirc, the variable is called `MesheryFolder`.

Reply:
Can't use that here. Import cycle issue.

Reply:
Ruh-roh. @metonymic-smokey, will you kindly open an issue to track this an item to fix once those constants have been centralized?